pt.yml

```diff
@@ -1,169 +1,175 @@
 pt:
-  activerecord:
-    errors:
-      models:
-        user:
-          attributes:
-            email:
-              invalid: "Entrar um email válido."
-              taken: "Desculpa, mas este email já foi registrado."
-            nickname:
-              taken: "Desculpa, mas este usuário já foi registrado."
-  network:
```

Comment:
I think that everything in the taking still needs to be nested under the network as that is what it's expecting